Dockerfile

```diff
@@ -16,36 +16,29 @@ RUN apk add --no-cache curl tar procps \
  && rm -f /tmp/apache-maven.tar.gz \
  && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
 
-RUN apk add dos2unix --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted
+ADD scripts/entrypoint.sh /
+
+RUN apk add dos2unix --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted \
+  && dos2unix /entrypoint.sh \
+  && apk del dos2unix
 
 ADD ./overrides/settings-docker.xml /usr/share/maven/ref/
 ADD overrides /overrides
-ADD scripts/entrypoint.sh /
-RUN dos2unix /entrypoint.sh
 
 # PYX
 ENV GIT_BRANCH master
 VOLUME /app /output
 
-# OVERRIDE:
-# Specify geoip2 version until https://github.com/ajanata/PretendYoureXyzzy/pull/228
-RUN apk --no-cache add git openssh \
+RUN apk --no-cache add git \
  && git clone -b $GIT_BRANCH https://github.com/ajanata/PretendYoureXyzzy.git /project \
  && cd project \
  && cp build.properties.example build.properties \
  && cp build.properties.example build.properties.a \
  && cat build.properties.a /overrides/build.properties > build.properties \
- && mvn versions:use-dep-version \
-  -Dincludes=com.maxmind.geoip2:geoip2 \
-  -DdepVersion=2.8.1 \
-  -DforceVersion=true \
  && mvn clean package war:war \
   -Dhttps.protocols=TLSv1.2 \
   -Dmaven.buildNumber.doCheck=false \
-  -Dmaven.buildNumber.doUpdate=false
-
-ENTRYPOINT ["/entrypoint.sh"]
+  -Dmaven.buildNumber.doUpdate=falsev
 
 WORKDIR /project
-
-CMD cat build.properties.a /overrides/build.properties > build.properties && mvn clean package war:exploded jetty:run -Dhttps.protocols=TLSv1.2 -Dmaven.buildNumber.doCheck=false -Dmaven.buildNumber.doUpdate=false
```

Comment:
If we remove this line that concats the volumed-in `build.properties` file, is there still a way for users to add their own overrides? There are a couple issues from people asking for this feature, just want to make sure that they're not left out.

Reply:
I guess I am confused because it seems that the same thing is done twice. Looking with the build and pull RUN we do the concat there as well as the build. Seems like we would only need to do it in one place. I will admit that my experience is not with java /maven so I could be wrong there.

Reply:
Oh interesting, nice catch! Kinda looks like the `RUN ... cp... cat ...` on L37 is left over from a refactor. `/overrides/` is only volumed in at runtime, not build time, right? That seems pointless during build.

Runtime overrides are needed (I think), not build-time.

I think we could remove L35-37, then we could move `cat build.properties.example /overrides/build.properties > build.properties` into `entrypoint.sh`. That would at least clean up CMD a bit and would still allow for customization. 👍

Reply:
Overrides is not being used as a volume but we are adding the values from overrides when the Docker image is built. I am looking into making it so overrides is added during run rather than build.

Reply:
My mistake then, I guess we need to keep it for build-time after all.

Reply:
I left it as run-time only. Didn't make sense to me to build it with one thing the override that at run time vs not at build time and overriding it during run.